README.md

```diff
@@ -63,3 +63,12 @@ In the image below you can find some score examples.
 More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
 
 ![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+
+### Reflections
+- I am not confident if I am always testing behaviour rather than state. Tried my best to minimise using eq matcher in rspec tests for this but not sure if using change_by is good enough if I'm referring to instance variables a lot.
```

Comment:
@alicelieutier do you mean that if a method is simply updating the value of an instance variable, that it would be better to create a getter method that retrieves the value and to test the output of the getter method as opposed to accessing the instance variable directly? Or is that enough?

Reply:
No, I mean use a public method that already exists that depend on that state. If there is none, then it would mean that changing the state has no effect whatsoever to the outside. Which means that state is not needed.

Reply:
Understood. Thanks!